langchain/src/chains/llm_chain.ts

```diff
@@ -26,7 +26,7 @@ export interface LLMChainInput extends ChainInputs {
   outputParser?: BaseOutputParser;
 
   /** @ignore */
-  outputKey: string;
+  outputKey?: string;
```

Comment:
why did this change?

Reply:
Because I'm now using this type in the constructor, and it's optional there